src/main/java/algorithms/minimumSpanningTree/prim/Prim.java

```diff
@@ -0,0 +1,76 @@
+package algorithms.minimumSpanningTree.prim;
```

Comment:
i believe the underlying assumption is that the graph is connected right? i think both ur implementation should work fine for unconnected graphs too (not a thorough analysis.. jsut a quick run-through in my head) - that is, your return matrix should capture separate MSTs, if exist. Do double check and lmk if im wrong!